hack/integration-benchmarks.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+source "${KUBE_ROOT}/hack/lib/init.sh"
+
+cleanup() {
+  kube::etcd::cleanup
+  kube::log::status "Benchmark cleanup complete"
+}
+
+runTests() {
+  # TODO: Start etcd in a cgroup, profiling is only useful when we
+  # can limit its cpu and memory.
+  kube::etcd::start
+  kube::log::status "Running benchmarks"
+  KUBE_GOFLAGS="-tags 'benchmark no-docker' -bench 'BenchmarkWatchPods.*' -benchtime 10s -cpu 1" \
```

Comment:
i guess, but i wasn't sure if we would overload shippable by running all our benchmarks in the background. It seems kind-of fickle.